pjsip/src/pjsua2/media.cpp

```diff
@@ -321,7 +326,8 @@ static pj_status_t get_frame(pjmedia_port *port, pjmedia_frame *frame)
 
 static pj_status_t put_frame(pjmedia_port *port, pjmedia_frame *frame)
 {
-    AudioMediaPort *mport = (AudioMediaPort *) port->port_data.pdata;
+    struct port_data *pdata = (struct port_data *)port->port_data.pdata;
+    AudioMediaPort *mport = pdata->mport;
```

Comment:
Note that `get/put_frame()` may be called after the `AudioMediaPort` instance is destroyed, so perhaps `mport` field need to be reset in `AudioMediaPort` destructor. Even perhaps better to protect all access to the field with (group) lock.

Reply:
Ouch, this is more complicated than I anticipated and never seems to be mentioned anywhere.
Perhaps need to add this in the PR desc: Due to the async nature of conf port removal, media port must be prepared to receive further get/put_frame() callbacks until the removal completes.

Reply:
Added to backward compatibility issue in PR & 2.15 release note.